ansible/deploy_slave.yml

```diff
@@ -31,8 +24,12 @@
         - python3-dev
         - python3-six
         - python-dev
+        - sudo
```

Comment:
I am not so sure about adding sudo at this point; I believe this problem occurred when ansible tried to become a different user? In that case, I would rather handle that dependency separately, just as python is (as you can not run ansible at all without python on the destination). However, given the "hackiness" of the rest of the playbooks, it should not be too much of a problem keeping it here for convenience.